src/app/app.component.ts

```diff
@@ -103,6 +109,7 @@ export class AppComponent {
 
   private async initializeApp() {
     this.platform.ready().then(async () => {
+      this.platforms = this.platform.platforms().join(" ");
```

Comment:
Originally I planned to use `Capacitor.getPlatform()`, which just returns "web", "ios" or "android". But there are two potential reasons to prefer `platform.platforms()`:
1. In the future we may want to to extend our platform specific CSS to apply rules based on platform types within this hierachy (e.g. "mobile")
2. When running on a web browser with device mode active, `Capacitor.getPlatform()` returns "web", but `platform.platforms()` returns a list representing the emulated device. E.g. in Chrome with device mode active and set to iPhone SE:
    <img width="423" alt="Screenshot 2024-05-08 at 16 06 57" src="https://github.com/IDEMSInternational/parenting-app-ui/assets/64838927/806138f9-7cc5-44d8-ab26-52c627dd3e94">

    - This seems desirable as it allows us to test functionality when running on web, although there may be issues with this that would make the simple platform more suitable

Reply:
Makes a lot of sense, also appreciate storing it as string instead of array to use with the css selectors and providing the option for various levels of platform-specific override.